notebooks/process_demetr_results.ipynb

```diff
@@ -0,0 +1,3162 @@
+{
```

Comment:
<p>Should be documented in the notebook/elsewhere where to get all the necessary input files, what scripts to run first etc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/ARC-m4st/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>